physicsnemo/utils/domino/utils.py

```diff
@@ -18,7 +18,7 @@
 Important utilities for data processing and training, testing DoMINO.
 """
 
-import os
```

Comment:
Note: in general, `pathlib` is preferred over `os.path`, since as far back as the Python 3.4 days:
- https://medium.com/@barila/choose-your-path-pythons-pathlib-vs-os-path-4de0b1e752dd
- https://www.reddit.com/r/learnpython/comments/i29h6j/when_or_why_should_i_use_pathlibpath_over_ospath/